pages/blog/beyond-boundaries.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Beyond Boundaries - Empowering Through OSS Mentorship'
+date: 2023-07-12T06:00:00+01:00
+type: Community
+tags:
+  - Mentorships
+  - OSS
+  - AsyncAPI
+cover: /img/posts/beyond.webp
+weight: 10
+authors:
+  - name: Azeez Elegbede
+    photo: /img/avatars/ace.webp
+    link: https://twitter.com/_acebuild
+    byline: AsyncAPI Preacher
+---
+
+In 2022 we piloted our own very innovative mentorship program that focuses on open-source software(OSS). We envisioned a platform where aspiring developers could learn, grow, and contribute to exciting projects while receiving guidance from experienced mentors.
+
+## Our Backstory
+
+At AsyncAPI, we value giving back to the community, as it gives us a sense of purpose as a community-first organization, and we are always looking for ways to participate in Opensource mentorship programs like Google Summer of Code(GSoC), Google Season of Docs(GSoD) and more because the fulfilling feeling of giving back and contributing to society gotten from this programs is unparalleled.
```

Comment:
```suggestion
At AsyncAPI, we value giving back to the community, giving us a sense of purpose as a community-first organization. We are always looking for ways to participate in open-source mentorship programs like Google Summer of Code(GSoC), Google Season of Docs(GSoD), and more because the fulfilling feeling of giving back and contributing to society gotten from this program is unparalleled.
```

Reply:
How about substituting "this program" for "these programs" in the last sentence?